pkg/kubelet/lifecycle/predicate.go

```diff
@@ -222,32 +207,19 @@ func (e *PredicateFailureError) GetReason() string {
 }
 
 // GeneralPredicates checks a group of predicates that the kubelet cares about.
-func GeneralPredicates(pod *v1.Pod, nodeInfo *schedulerframework.NodeInfo) ([]PredicateFailureReason, error) {
-	if nodeInfo.Node() == nil {
-		return nil, fmt.Errorf("node not found")
-	}
-
+func generalPredicates(pod *v1.Pod, nodeInfo *schedulerframework.NodeInfo) []PredicateFailureReason {
```

Comment:
let's take the chance to rename this to `generalFilters` or even `nodeFilters`

Reply:
maybe `generalFilters ` is better.